src/components/signin/reducers.js

```diff
@@ -14,7 +14,7 @@ export default (state = initialState, { type, payload }) => {
         case TYPES.SET_UNAUTHENTICATED:
             localStorage.removeItem('FBToken');
             localStorage.removeItem('persist:root');
```

Comment:
is there a reason why you are removing  `persist:root` ?

Reply:
persist:root stores the entire store on the user's local storage, i felt it was okay to remove such from local storage on logout